image_explorer/image_explorer.py

```diff
@@ -201,6 +211,7 @@ def studio_view(self, context):
     def studio_submit(self, submissions, suffix=''):
 
         self.display_name = submissions['display_name']
+        self.hotspot_coordinates_centered = submissions['hotspot_coordinates_centered']
```

Comment:
We should only ever allow going from the old deprecated coordinate system to the new centered system, so please replace this line with:

```python
if submissions.get('hotspot_coordinates_centered', False):
    self.hotspot_coordinates_centered = True
```